renovate-config.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Default configuration for repositories in the Laminas organisation",
+    "extends": [
+        ":enableRenovate",
+        ":ignoreModulesAndTests",
+        ":automergeMinor",
+        ":automergeBranch",
+        ":prNotPending",
+        ":rebaseStalePrs",
+        ":separateMajorReleases",
+        ":combinePatchMinorReleases",
+        ":enableVulnerabilityAlerts",
+        ":timezone(UTC)",
+        ":label(renovate)",
+        "group:allNonMajor",
+        "schedule:nonOfficeHours",
+        "github>whitesource/merge-confidence:beta"
```

Comment:
```suggestion
```

Renovate enables this by default on the hosted app. [^docs] If you're self-hosting Renovate, you should keep this line to have the Merge Confidence badges.

> ## Enabling and Disabling
> 
> Merge Confidence badges are enabled automatically for anyone using the WhiteSource Renovate App. If you don't use the app, you can enable Merge Confidence badges by adding this to your Renovate config:
> 
> ```json
> "extends": ["github>whitesource/merge-confidence:beta"]
> ```
> 
> If you’d prefer to disable these badges in the app, add this to your config instead:
> 
> ```
> "ignorePresets": ["github>whitesource/merge-confidence:beta"]
> ```

[^docs]: https://docs.renovatebot.com/merge-confidence/#enabling-and-disabling

Reply:
Thanks, that's good to know. I'll leave this unresolved for now as a decision still needs to be made by the TSC as to whether to use the app or self-hosted. Once that decision is made I can commit/reject the change as needed.

Reply:
My comment only applies if you're extending from `config:base`. 🙈 

You're not extending from `config:base` in this PR, so if you wanted to have the Confidence Badges you'd need to explicitly opt-in, just like you're already doing now. 😉

Reply:
Ah, even if using the hosted app?

Reply:
> Ah, even if using the hosted app?

I'm getting mixed up with Dependency Dashboard and the merge confidence badge, sorry about that!

---

To clarify, this is what you'll get if you were to use the `config:base` preset: [^config-base]

```json
{
  "extends": [
    ":dependencyDashboard",
    ":semanticPrefixFixDepsChoreOthers",
    ":ignoreModulesAndTests",
    ":autodetectPinVersions",
    ":prHourlyLimit2",
    ":prConcurrentLimit10",
    "group:monorepos",
    "group:recommended",
    "workarounds:all"
  ]
}
```

As you can see `config:base` does enable the Dependency Dashboard (like I said in another comment).

---

The Merge Confidence badge follows these docs [^docs], which means its enabled by default when you use the hosted app.

> ## Enabling and Disabling
> 
> Merge Confidence badges are enabled automatically for anyone using the WhiteSource Renovate App. If you don't use the app, you can enable Merge Confidence badges by adding this to your Renovate config:
> 
> ```json
> "extends": ["github>whitesource/merge-confidence:beta"]
> ```
> 
> If you’d prefer to disable these badges in the app, add this to your config instead:
> 
> ```
> "ignorePresets": ["github>whitesource/merge-confidence:beta"]
> ```

[^docs]: https://docs.renovatebot.com/merge-confidence/#enabling-and-disabling

[^config-base]: https://docs.renovatebot.com/presets-config/#configbase

Reply:
Ok, thanks for clarifying.

Reply:
I've committed this change.